bot.py

```diff
@@ -39,6 +53,10 @@ async def on_ready():
         await bot.add_cog(Uptime(bot))
         await bot.add_cog(ServerInfo(bot))
         await bot.add_cog(Logging(bot))
+        # Uncomment for using ConfigData COG
```

Comment:
I think we do want to use it already?

Reply:
Can be used, it doesn't depend on othe Cogs